js/ui/context_menu/ui.menu_base.js

```diff
@@ -177,18 +180,44 @@ class MenuBase extends HierarchicalCollectionWidget {
         this._initActions();
     }
 
+    _getLinkContainer(iconContainer, textContainer, { linkAttr, url }) {
+        if(iconContainer) {
+            iconContainer.addClass(DX_ICON_WITH_URL_CLASS);
```

Comment:
it's more compact to use optional chaining in such cases
```suggestion
            iconContainer?.addClass(DX_ICON_WITH_URL_CLASS);
```